jwst/extract_1d/ifu.py

```diff
@@ -164,6 +170,10 @@ def ifu_extract1d(input_model, ref_file, source_type, subtract_background,
     del npixels_temp
     del npixels_bkg_temp
 
+    temp_flux_rf = None
+    surf_bright_rf = None
+    background_rf = None
+
     # If selected, apply 1d residual fringe correction to the extracted spectrum
     if ((input_model.meta.instrument.name == 'MIRI') & (extract_params['ifu_rfcorr'] is True)):
```

Comment:
This should be the logical `and`, not the bitwise `&`

```suggestion
    if ((input_model.meta.instrument.name == 'MIRI') and (extract_params['ifu_rfcorr'] is True)):
```